iceoryx_hoofs/test/moduletests/test_cxx_variant.cpp

```diff
@@ -193,12 +193,11 @@ TEST_F(variant_Test, EmplaceSecondValidElementWorks)
     EXPECT_THAT(sut.get<ComplexClass>()->b, Eq(65.03F));
 }
 
-TEST_F(variant_Test, DISABLED_emplaceInvalidElement)
+TEST_F(variant_Test, EmplaceInvalidElementCompileTimeCheck)
 {
     ::testing::Test::RecordProperty("TEST_ID", "6fa3b290-8249-4825-8eac-72235a06710e");
-    // this is a compile time check, if you uncomment this the compiler will
-    // fail
-    //    EXPECT_THAT(sut.emplace< unsigned int >(0), Eq(false));
+    GTEST_SKIP() << "Compile time check, if you uncomment this test, the compiler will fail";
```

Comment:
You broke your own rule here :smirk: 

Where is the issue number :smile:

Reply:
I would be glad if you can help me finding the right ticket for this particular case. I'm also fine with removing that test.